scripts/deploy/nlp/deploy_triton.py

```diff
@@ -52,6 +53,8 @@ def get_args(argv):
         description=f"Deploy nemo models to Triton",
     )
     parser.add_argument("-nc", "--nemo_checkpoint", type=str, help="Source .nemo file")
+    parser.add_argument("-hfp", "--hf_model_path", type=str, help="Huggingface model path")
```

Comment:
Can we use only one parameter for the hf model like here? https://github.com/NVIDIA/NeMo/blob/onur/hf-inframework-deploy/scripts/deploy/nlp/deploy_inframework_hf_triton.py#L67